src/content/guides/guide-connect-supabase.md

```diff
@@ -50,13 +44,9 @@ Svelte,https://svelte.dev/,svelte.svg,0,"Svelte is a UI framework that uses a co
 
 This will give you a preview of the data that will be inserted into the database. Click **Import data** to add the data to the database.
 
-## Install the Supabase Netlify extension
-
-Now we can install the [Supabase extension](https://app.netlify.com/extensions/supabase). In the Netlify UI, go to your team’s dashboard, navigate to **Extensions** and click on the Supabase extension. Click the install button to install the extension.
-
-### Configure the Supabase extension
+## Configure the Supabase Netlify extension
 
-After the extension is installed, navigate to the Supabase template site that you deployed, and go to **Site configuration**. In the **General** settings, you will see a new **Supabase** section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
+The [Netlify Supabase extension](https://app.netlify.com/extensions/supabase) should already be installed. Visit your site's configuration page and scroll to the Supabase section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
```

Comment:
```suggestion
The [Supabase Netlify extension](https://app.netlify.com/extensions/supabase) should already be installed. Visit your site's configuration page and scroll to the Supabase section. Click **Connect** to connect your Netlify site to your Supabase account using OAuth.
```
Sorry this was my copy error - let's match the title as I think that's the correct name